src/js/release_loader.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const ReleaseLoader = function (url) {
+    this._url = url;
+    this._cacheExpirationPeriod = 3600 * 1000;
+};
+
+ReleaseLoader.prototype.load = function (url, onSuccess, onFailure) {
+
+    const dataTag = `${url}_Data`;
+    const cacheLastUpdateTag = `${url}_LastUpdate`;
+
+    const result = SessionStorage.get([cacheLastUpdateTag, dataTag]);
+    const dataTimestamp = $.now();
+    const cachedData = result[dataTag];
+    const cachedLastUpdate = result[cacheLastUpdateTag];
+
+    const cachedCallback = () => {
+        if (cachedData) {
+            GUI.log(i18n.getMessage('buildServerUsingCached', [url]));
+        }
```

Comment:
Why not using `class` here? `prototype` hacking is sort of thing of the past for new code.